domains/surya0.json

```
@@ -0,0 +1,13 @@
+{
+        "owner": {
+           "username": "suryapratapsinghsuryavanshi",
+           "email": "",
+           "discord": "858030091106517042",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.DS_lQ-rkLkv0VFei9VOKq8YDnykOlXX8Rvlzzp8KSlTpDqnRY8e3X1_J4iVfI9t6KJdEfFjnR1r3xhXC9I2A2khHqZMlQTgdhyi72OLHDIQPur3lC16oZlz1T3d5BrVUdKd8ueNky_OLyIZDW9bYY_uR-8vy7ahqasWUW-EEdBspfeMDzWlUon7NKqZaR79maAArOCnoSjDg4Ul5vVei5w-EarI99nylk2tsk8Nx2KyjzEPxx_l7WLUWtZ8LOBRsV83KjJTKRtGZd0eD4jolAW6OQDnD9JEoR8g2XKVEylaYQlHJNgWLdHNdCdJcwhG6Ge2IRkvtlvhwQDu5230Mbg.4c2jfDHUTjsiJZZD2GNNWA.c7WmV_BgqxX8VhuhOyFxZJurynqJ4skcvPWenpEsKXrLG5j93rzAlh-kjG6A6ZIQpEu2NHwKEzvRu_S_PwijC-Ctq5GxiM8rCQfyvuAif4F-kroGiCPD0092yKmBa2kn.Ljp9qlE0JjM35gTNNQQPRQ"
+        },
+    
+        "record": {
+            "CNAME": "https://suryapratap.netlify.app/"
```

CNAME can not contain `https://` or `/`
```suggestion
            "CNAME": "suryapratap.netlify.app"
```